pkg/template/servicebroker/lastoperation.go

```diff
@@ -35,6 +35,21 @@ func (b *Broker) LastOperation(instanceID string, operation api.Operation) *api.
 		return api.InternalServerError(err)
 	}
 
+	namespace := brokerTemplateInstance.Spec.TemplateInstance.Namespace
+
+	//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, remove the user name empty string checks
+	if u.GetName() != "" {
+		if err := util.Authorize(b.kc.Authorization().SubjectAccessReviews(), u, &authorization.ResourceAttributes{
+			Namespace: namespace,
+			Verb:      "get",
+			Group:     templateapi.GroupName,
+			Resource:  "templateinstances",
+			Name:      brokerTemplateInstance.Spec.TemplateInstance.Name,
+		}); err != nil {
+			return api.Forbidden(err)
+		}
+	}
+
 	templateInstance, err := b.templateclient.TemplateInstances(brokerTemplateInstance.Spec.TemplateInstance.Namespace).Get(brokerTemplateInstance.Spec.TemplateInstance.Name, metav1.GetOptions{})
```

Comment:
please can you do a replace brokerTemplateInstance.Spec.TemplateInstance.Namespace -> namespace here and wherever else you can in this package?